.github/actions/common-setup-ec2-linux/action.yml

```diff
@@ -14,18 +14,12 @@ runs:
     - name: Setup Linux
       uses: ./.github/actions/setup-linux
     - name: Chown workspace
```

Comment:
@malfet We don't need chown workspace steps anymore after your change, yes?

These could just be erased.